views/overview/pod_panel.go

```diff
@@ -91,134 +99,171 @@ func (p *podPanel) DrawBody(data interface{}) {
-				Text:  pod.Namespace,
-				Color: tcell.ColorYellow,
-				Align: tview.AlignLeft,
-			},
-		)
-
-		p.list.SetCell(
-			i, 1,
-			&tview.TableCell{
-				Text:  pod.Name,
-				Color: tcell.ColorYellow,
-				Align: tview.AlignLeft,
-			},
-		)
-
-		p.list.SetCell(
-			i, 2,
-			&tview.TableCell{
-				Text:  fmt.Sprintf("%d/%d", pod.ReadyContainers, pod.TotalContainers),
-				Color: tcell.ColorYellow,
-				Align: tview.AlignLeft,
-			},
-		)
-
-		p.list.SetCell(
-			i, 3,
-			&tview.TableCell{
-				Text:  pod.Status,
-				Color: tcell.ColorYellow,
-				Align: tview.AlignLeft,
-			},
-		)
-
-		p.list.SetCell(
-			i, 4,
-			&tview.TableCell{
-				Text:  fmt.Sprintf("%d", pod.Restarts),
-				Color: tcell.ColorYellow,
-				Align: tview.AlignLeft,
-			},
-		)
-
-		p.list.SetCell(
-			i, 5,
-			&tview.TableCell{
-				Text:  pod.TimeSince,
-				Color: tcell.ColorYellow,
-				Align: tview.AlignLeft,
-			},
-		)
-
-		// Volume
-		p.list.SetCell(
-			i, 6,
-			&tview.TableCell{
-				Text:  fmt.Sprintf("%d/%d", pod.Volumes, pod.VolMounts),
-				Color: tcell.ColorYellow,
-				Align: tview.AlignLeft,
-			},
-		)
-
-		p.list.SetCell(
-			i, 7,
-			&tview.TableCell{
-				Text:  pod.IP,
-				Color: tcell.ColorYellow,
-				Align: tview.AlignLeft,
-			},
-		)
-
-		p.list.SetCell(
-			i, 8,
-			&tview.TableCell{
-				Text:  pod.Node,
-				Color: tcell.ColorYellow,
-				Align: tview.AlignLeft,
-			},
-		)
-
-		if metricsDisabled {
-			cpuRatio = ui.GetRatio(float64(pod.PodRequestedCpuQty.MilliValue()), float64(pod.NodeAllocatableCpuQty.MilliValue()))
-			cpuGraph = ui.BarGraph(10, cpuRatio, colorKeys)
-			cpuMetrics = fmt.Sprintf(
-				"[white][%s[white]] %dm %02.1f%%",
-				cpuGraph, pod.PodRequestedCpuQty.MilliValue(), cpuRatio*100,
-			)
-
-			memRatio = ui.GetRatio(float64(pod.PodRequestedMemQty.MilliValue()), float64(pod.NodeAllocatableMemQty.MilliValue()))
-			memGraph = ui.BarGraph(10, memRatio, colorKeys)
-			memMetrics = fmt.Sprintf(
-				"[white][%s[white]] %dGi %02.1f%%", memGraph, pod.PodRequestedMemQty.ScaledValue(resource.Giga), memRatio*100,
-			)
-		} else {
-			cpuRatio = ui.GetRatio(float64(pod.PodUsageCpuQty.MilliValue()), float64(pod.NodeAllocatableCpuQty.MilliValue()))
-			cpuGraph = ui.BarGraph(10, cpuRatio, colorKeys)
-			cpuMetrics = fmt.Sprintf("[white][%s[white]] %dm %02.1f%%", cpuGraph, pod.PodUsageCpuQty.MilliValue(), cpuRatio*100)
-
-			memRatio = ui.GetRatio(float64(pod.PodUsageMemQty.MilliValue()), float64(pod.NodeUsageMemQty.MilliValue()))
-			memGraph = ui.BarGraph(10, memRatio, colorKeys)
-			memMetrics = fmt.Sprintf("[white][%s[white]] %dMi %02.1f%%", memGraph, pod.PodUsageMemQty.ScaledValue(resource.Mega), memRatio*100)
+	for rowIdx, pod := range pods {
+		rowIdx++ // offset for header row
+		
+		// Render each column that is included in the filtered view
+		for _, colName := range p.listCols {
+			colIdx, exists := p.colMap[colName]
+			if !exists {
+				continue
+			}
+			
+			switch colName {
+			case "NAMESPACE":
```

Comment:
Suggestion: store each column name in a constant var. That way the rest of the code simply refer to the name instead of specifying the name value each time.